source/options.html

```diff
@@ -23,6 +23,13 @@ <h2 class="only-firefox">Options</h2>
 		</label>
 	</p>
 
+	<p>
+		<label>
+			<strong>Personal Token</strong> (for high rate API calls and private repos)<br>
+			<input type="password" name="personalToken"></textarea>
```

Comment:
Set `autocomplete="off"` attribute.

And if it's correct that tokens are always `40`, we should set `size="40"`.